packages/compass-e2e-tests/helpers/result-logger.ts

```diff
@@ -58,25 +60,77 @@ const ENV_VARS = [
   'RUNNER_OS',
   'RUNNER_TEMP',
   'RUNNER_TOOL_CACHE',
-];
+] as const;
 
-function joinPath(parts) {
+type Env = { [K in typeof ENV_VARS[number]]?: string };
+
+type HookOrTest = Mocha.Hook | Mocha.Test;
+
+enum ResultStatus {
+  Start = 'start',
+  Pass = 'pass',
+  Fail = 'fail',
+  SilentFail = 'silentfail',
+}
+
+enum CI {
+  Evergreen = 'evergreen',
+  GithubActions = 'github-actions',
+  Unknown = 'unknown',
+}
```

Comment:
Do we need enums here or would `type ResultStatus = 'start' | 'pass' | 'fail' | 'silentfail';` also do the trick? Almost all of the time enums only provide an additional layer of indirection without any additional value

Reply:
You're right. Will change it.